cordyceps/Cargo.toml

```diff
@@ -18,6 +18,7 @@ categories = ["data-structures", "no-std"]
 default = []
 alloc = []
 std = ["alloc"]
+no-cache-pad = []
```

Comment:
maybe worth a comment explaining what this does here? and/or documenting it in the `lib.rs` docs? but, we don't currently have a list of other feature flags, so it would be fine to add that in a separate PR...